src/MongoDB.Driver/MongoUrl.cs

```diff
@@ -69,6 +69,7 @@ public class MongoUrl : IEquatable<MongoUrl>
         private readonly TimeSpan _socketTimeout;
         private readonly string _username;
         private readonly bool _useTls;
+        private readonly bool _tlsDisableCertificateRevocationCheck;
```

Comment:
Keep private fields in alphabetical order.

Reply:
Fixed, sorry, I was all backwards when I placed the properties :)